core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/split.kt

```diff
@@ -62,22 +66,27 @@ public typealias ColumnNamesGenerator<C> = ColumnWithPath<C>.(extraColumnIndex:
 
 // region default
 
+@Interpretable("SplitDefault")
 public inline fun <T, C : Iterable<R>, reified R> Split<T, C>.default(value: R?): SplitWithTransform<T, C, R> =
     by { it }.default(value)
 
+@Deprecated(SPLIT_STR, ReplaceWith("""by(",").default(value)"""))
```

Comment:
`ReplaceWith` works better with explicit `this.` in it. Also, The text inside it can be put in the deprecations.kt file as well if you like.

Reply:
Also, it used to be done with `splitDefault()`, which does a trim as well: `split(",").map { it.trim() }`